### PR TITLE
Add: pyospackage recipe to conda-forge (pyOpenSci)

### PR DESCRIPTION
**Pull request by lwasser**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/pyospackage/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  imports:
+    - pyospackage
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  summary: A package that adds numbers together
+  dev_url: https://github.com/pyopensci/pyospackage
```
Comment (lwasser) [inline recipes/pyospackage/meta.yaml:35]:
```suggestion
  home: https://pypi.org/project/pyospackage/
  summary: A package that adds numbers together
  dev_url: https://github.com/pyopensci/pyospackage
```

**Comment by lwasser** [comment]
hey there @ocefpaf  👋 this is the pyOpenSci package I was telling you about. For some reason it's failing windows now. i'm open to making changes but i'm not sure where to start. it's a fairly simple package with no dependencies.  many thanks!

File: recipes/pyospackage/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+test:
+  imports:
+    - pyospackage
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://pypi.org/project/pyospackage/
```
Comment (lwasser) [inline recipes/pyospackage/meta.yaml:34]: i had to manually add this as grayskull seems to not add this element no matter what is in the pyproject.toml file.

**Comment by ocefpaf** [comment]
Hey @lwasser, I updated the branch b/c the failure was in a dangling package, not your recipe. Things should pass now. Also, I see you already opened an issue for grayskull. I'll check it upstream ASAP.

**Review by ocefpaf — APPROVED** [review]
@lwasser the recipe is perfect. My guess is that you don't want to merge this, right? But, if you do, it passes the review process and should pass the CIs. Let me know if you need anything else.

**Comment by lwasser** [comment]
@ocefpaf if it is ok to merge this, i'd love to do that. i'll then add a section on how to maintain a conda-forge feedstock and i can do a demo that shows how the pypi - conda-forge connection works on the user end.

**Comment by lwasser** [comment]
Also thank you for reviewing and fixing the pr! grayskull just don't add a home: key:value pair in my yaml file no matter what is in my pyproject.toml file. so i added it manually as the linter here required it.

**Pull request merged**